testing/bird/APKBUILD

```diff
@@ -32,49 +32,39 @@ _build() {
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
 		--localstatedir=/var \
-		$@ \
-		|| return 1
-	make || return 1
+		$@
+	make
 }
 
-
 build() {
 	cd "$builddir"
 	mkdir -p ipv6 ipv4
-	cd "$builddir"/ipv6
-	_build --enable-ipv6 || return 1
-	cd "$builddir"/ipv4
-	_build || return 1
+	cd "$builddir/ipv6"
```

Comment:
Same as above, I'm not a big fan of the unnecessary quoting, although it's not a huge deal here. Existing style is to only quote variables, not the entire thing that includes the variable.